src/utils/dynamoDb.js

```diff
@@ -6,7 +6,7 @@ const createDynamoDbInstance = () => new AWS.DynamoDB({
 });
 
 const convertToDynamoDbRecord = (data) => AWS.DynamoDB.Converter.marshall(
-  data, { convertEmptyValues: true },
+  data, { convertEmptyValues: false },
```

Comment:
`convertEmptyValues` can not parse empty object that has to be set as `config` in the initial plot config.